src/templates/user/settings/financial_assessmentws.html.tt

```diff
@@ -0,0 +1,287 @@
+                        </div>
+                        <div class="gr-8">
+                            <select id="net_income" name="net_income">
+                                <option value="">Please select</option>
+                                <option value="Less than &#36;25,000">Less than &#36;25,000</option>
+                                <option value="&#36;25,000 - &#36;50,000" >&#36;25,000 - &#36;50,000</option>
+                                <option value="&#36;50,001 - &#36;100,000">&#36;50,001 - &#36;100,000</option>
+                                <option value="&#36;100,001 - &#36;500,000">&#36;100,001 - &#36;500,000</option>
+                                <option value="Over &#36;500,000">Over &#36;500,001</option>
+                            </select>
+                            <p class="errorfield" id="errornet_income"></p>
+                        </div>
+                    </div>
+                    <div class="gr-padding-10 gr-row clear">
+                        <div class="gr-4  form_label">
+                            <label for="estimated_worth">Estimated Net Worth</label>
+                        </div>
+                        <div class="gr-8">
+                            <select id="estimated_worth" name="estimated_worth">
+                                <option value="">Please select</option>
+                                <option value="Less than &#36;100,000">Less than &#36;100,000</option>
+                                <option value="&#36;100,000 - &#36;250,000" >&#36;100,000 - &#36;250,000</option>
+                                <option value="&#36;250,001 - &#36;500,000">&#36;250,001 - &#36;500,000</option>
+                                <option value="&#36;500,001 - &#36;1,000,000">&#36;500,001 - &#36;1,000,000</option>
+                                <option value="Over &#36;1,000,000">Over &#36;1,000,000</option>
+                            </select>
+                            <p class="errorfield" id="errorestimated_worth"></p>
+                        </div>
+                    </div>
+                </fieldset>
+                <span class="tl">&nbsp;</span><span class="tr">&nbsp;</span><span class="bl">&nbsp;</span><span class="br">&nbsp;</span>
+            </div>
+        </div>
+        <div class="rbox form">
+            <div class="rbox-wrap">
+                <fieldset>
+                    <div class="gr-12">
+                        <p id="warning">The financial trading services contained within this site are only suitable for customers who are able to bear the loss of all the money they invest and who understand and have experience of the risk involved in the acquistion of financial contracts. Transactions in financial contracts carry a high degree of risk. If purchased contracts expire worthless, you will suffer a total loss of your investment, which consists of the contract premium.</p>
```

Comment:
remove the sentence "If purchased contracts expire worthless, you will suffer a total loss of your investment, which consists of the contract premium."

Indeed, contracts in MT5 cannot "expire worthless" - that's a concept that only applies to binaries.